tests/model_executor/test_model_load_with_params.py

```diff
@@ -46,8 +46,7 @@ def test_model_loading_with_params(vllm_runner):
         assert model_tokenizer.tokenizer_config["do_lower_case"]
         assert model_tokenizer.tokenizer.model_max_length == 512
 
-        model = model.model.llm_engine.model_executor\
-                     .driver_worker.model_runner.model
+        model = model.get_torch_model()
```

Comment:
with the function input for `collective_rpc` , this can be achieved by:

```python
    def check_model(self):
        model = self.model_runner.model
        assert isinstance(model, BertEmbeddingModel)
        assert model._pooler.pooling_type == PoolingType.CLS
        assert model._pooler.normalize
    assert all(model.model.collective_rpc(check_model))
```

do I miss anything?